plans/test_playbooks_parallel.fmf

```diff
@@ -0,0 +1,110 @@
+summary: A general test for a system role
+tag: playbooks_parallel
+provision:
+  # TF uses `how: artemis`, and `tmt try`` uses `how: virtual`.
+  # Hence there is no need to define `how` explicitly.
+  - name: control-node1
+    role: control_node
+    # `connection: system` is required for `how: virtual` to assign VMs a real
+    # IP making SSH configuration easier.
+    # This setting is ignored in `artemis`, so we can leave it as is.
+    connection: system
+  - name: managed-node1
+    role: managed_node
+    connection: system
+    hardware:
+      network:
+        - type: eth
+        - type: eth
+  - name: managed-node2
+    role: managed_node
+    connection: system
+    hardware:
+      network:
+        - type: eth
+        - type: eth
+  - name: managed-node3
+    role: managed_node
+    connection: system
+    hardware:
+      network:
+        - type: eth
+        - type: eth
+  - name: managed-node4
+    role: managed_node
+    connection: system
+    hardware:
+      network:
+        - type: eth
+        - type: eth
+  - name: managed-node5
+    role: managed_node
+    connection: system
+    hardware:
+      network:
+        - type: eth
+        - type: eth
+  - name: managed-node6
+    role: managed_node
+    connection: system
+    hardware:
+      network:
+        - type: eth
+        - type: eth
+environment:
+  ANSIBLE_VER: 2.17
+  REPO_NAME: network
+  PYTHON_VERSION: 3.12
+  SYSTEM_ROLES_ONLY_TESTS: "tests_default.yml"
```

Comment:
@spetrosi is this just for testing this PR?

Reply:
A leftover from my local tests, removed it. Testing a PR is not possible unfortunately.